docs/change-log/index.md

```diff
@@ -13,16 +14,183 @@
+- Supports SQLSource for batch ingestion. See [SQL data source](../data-prep/ingest-data-fs.html#sql-data-source).
+- Supports SQLTarget for storey engine. (Spark is not yet supported.) See [SQL target store](../data-prep/ingest-data-fs.html#sql-target-store).
+- Supports Spark using Redis as an online KV target, which cause a breaking change. See [Breaking changes](#breaking-changes).
+- The username and password for the RedisNoSqlTarget are now configured using secrets, as `<prefix_>REDIS_USER <prefix_>REDIS_PASSWORD` where \<prefix> is the optional RedisNoSqlTarget `credentials_prefix` parameter. See [Redis target store](../data-prep/ingest-data-fs.html#redis-target-store).
+- Offline data can be registered as feature sets. See [Create a feature set without ingesting its data](../feature-store/feature-sets.html#create-a-feature-set-without-ingesting-its-data).
+- `get_offline_features` supports Spark Operator and Remote Spark.
+
+#### Projects
+
+- When defining a new project from scratch, there is now a default `context` directory: `./`, which is the directory that the MLRun 
+    client runs from unless otherwise specified.  
+
+#### Serving graphs
+
+- Add support for graphs that split and merge (DAG), including a list of steps for the `after` argument in the `add_step()` method. See [Graph that splits and rejoins](../serving/model-serving-get-started.html#graph-that-splits-and-rejoins).
+- Supports configuring of consumer group name for steps following QueueSteps. See [Queue (streaming)](../serving/model-serving-get-started.html#queue-streaming).
+
+#### Storey
+- The event time in storey events is now taken from the `timestamp_key`. If the `timestamp_key` is not defined for the event, then the 
+    time is taken from the processing-time metadata. [View in Git](https://github.com/mlrun/mlrun/pull/2660), and in 
+    [Storey git](https://github.com/mlrun/storey/pull/394).
+
+#### UI
+- The new **Projects** home page provides easy and intuitive access to the full project lifecycle in three phases, with links to the 
+relevant wizards under each phase heading: ingesting and processing data, developing and training a model, deploying and monitoring the project.
+
+<p align="center"><img src="../_static/images/project-homepage.png" alt="mlrun-project-homepage" /></p><br>
+
+<a id="api-130"></a>
+### Deprecated and removed APIs
+Starting with v1.3.0, and continuing in subsequent releases, obsolete functions are getting removed from the code.
+
+**Deprecated and removed from v1.3.0 code**<br>
+These MLRun APIs have been deprecated since at least v1.0.0 and were removed from the code:
+
+| Deprecated/removed                   | Use instead                                   |
+| ------------------------------------ | --------------------------------------------- |
+| `project.workflows`                  | `project.spec.workflows`, such as `get_or_create_project`, `load_project` |
```

Comment:
You should remove the "such as..." part, it's not relevant here.

Reply:
what about this one?

Reply:
it was deleted